src/Microsoft.Extensions.Logging.Testing/LoggedTest/SinklessLoggedTest.cs

```diff
@@ -9,10 +9,10 @@
 
 namespace Microsoft.Extensions.Logging.Testing
 {
-    public abstract class LoggedTest
+    public class SinklessLoggedTest : ILoggedTest
```

Comment:
BaseLoggedTest? Other names? I kept the LoggedTest class as is to reduce churn in our code base.

Reply:
The TestSink is removed on this type so we can reduce the memory used by the tests cc @Tratcher .

Reply:
It's unfortunate that a type with the nice name would be the wrong one to use most of the time.

Reply:
I don't think it's wrong to use LoggedTest. In fact I would imagine it should be the one you would use most of the time. You would only switch to SinklessLoggedTest if you are using a large amount of memory.

Reply:
You should use it only when asserting logs, there is no point in wasting resources otherwise.

Reply:
Right, but that would mean I need to update all the repos that currently rely on LoggedTest for asserting logs. Not sure if it's really worth the effort to update them all.